src/pe_reports/report_generator.py

```diff
@@ -1,14 +1,30 @@
-"""A tool for creating Posture & Exposure reports."""
+"""A tool for creating Posture & Exposure reports.
+
+Usage:
+    pe-reports REPORT_DATE  DATA_DIRECTORY OUTPUT_DIRECTORY
```

Comment:
Remove unnecessary extra space character
```suggestion
    pe-reports REPORT_DATE DATA_DIRECTORY OUTPUT_DIRECTORY
```

Reply:
Resolved in 1299aba